SonarQube.MSBuild.Tasks/packages.config

```diff
@@ -1,7 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Build" version="14.3.0" targetFramework="net45" />
-  <package id="Microsoft.Build.Framework" version="14.3.0" targetFramework="net45" />
-  <package id="Microsoft.Build.Utilities.Core" version="14.3.0" targetFramework="net45" />
-    <package id="SonarAnalyzer.CSharp" version="6.6.0.3969" targetFramework="net45" developmentDependency="true" />
+  <package id="Microsoft.Build" version="14.3.0" targetFramework="net461" />
```

Comment:
They should all be `net46` and not `net461`. Shouldn't be a big issue if kept as is.